src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/EmployeesController.java

```diff
@@ -26,7 +26,7 @@
 
 @Controller
 @NextController(IntangibleAssetsNoteController.class)
-@PreviousController(EmployeesQuestionController.class)
+@PreviousController(OtherAccountingPolicyController.class)
```

Comment:
The `EmployeesController` class should no longer implement `ConditionalController`; we will always render the page going forward.